packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1589,7 +1594,7 @@ async function handler(
         resultBooking["paymentUid"] = payment?.uid;
       }
 
-      resultBooking = { ...foundBooking, seatReferenceUid: evt.attendeeSeatId };
+      resultBooking = { ...resultBooking, seatReferenceUid: evt.attendeeSeatId };
```

Comment:
This spread operator didn't consider the `paymentUid` being added in the previous step.